cx-sig-charter-draft.md

```diff
@@ -0,0 +1,110 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter adheres to the conventions described in the [Spinnaker Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The [Contributor Experience Special Interest Group] (SIG) is responsible for improving the experience of those who upstream contribute to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
```

Comment:
I would switch this `those who upstream contribute` to `those who contribute upstream`